Gemfile

```diff
@@ -1,22 +1,17 @@
 source 'https://rubygems.org'
 
 group :lint do
```

Comment:
This whole file can probably go away. Everything minus kitchen-docker is in ChefDK and that's not being used here actually